modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -16,139 +17,142 @@ The app monitoring dashboard helps you answer questions about your integrated ap
 
-. In Runtime Manager, in the menu on the left, click *Servers*. 
-. Click the row for the server to open the management panel on the right side of the page.
-. Click the *View Dashboard* button. +
-The dashboard for the server is displayed.
+To access the server monitoring dashboard:
 
+. In Runtime Manager, click *Servers*. 
+. Click the name of the server to monitor. 
++
+The dashboard for the server appears:
++
 image::monitoring-dashboards-server.png[]
 
-You can also navigate to a server's dashboard if you first open the dashboard for a cluster or server group that contains the server, then click the name of the server.
+You can also navigate to a server's monitoring dashboard from the <<Access the Server Group or Cluster Dashboard,server group or cluster dashboard>> that includes the server.
 
-== View Server Metrics
+=== View Server Metrics
 
-The monitoring dashboard for servers displays several metrics, which are divided into two panes:
+The server monitoring dashboard displays metrics divided into two panes:
 
-* *Overview*: View memory and CPU usage for the server at a high level.
-* *Breakdown*: View detailed metrics that track each part of the server's memory separately.
+* *Overview*:
++
+View memory and CPU usage for the server at a high level.
+* *Breakdown*
++
+View detailed metrics that track each part of the server's memory separately.
 
-To view graphs for a specific time period, select the time interval on the top-right of the page.
+To view graphs for a specific period, select the time interval on the top-right of the page:
 
 image::monitoring-dashboards-cbd92.png[]
 
-Hover your mouse over a given point of a graph, to display an overlay of numerical detail about activity on that instant.
+To display an overlay of numerical detail about activity that occurred at a specific time, hover your mouse over the graph:
 
 image::monitoring-dashboards-hover.png[]
 
-
-If your server runs multiple applications, you can select applications in the list to view the performance of these in a separate set of charts. To do so, select the application and a side-panel will display on the right with this data.
+If your server runs multiple applications, you can view the performance of each in a separate set of charts. To do so, select the application checkboxes to display the details pane on the right with this data.
 
 === Server Metrics
 
-The graphs in this server's dashboard cover different metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab, or you can view the *Breakdown* to see a detailed series of metrics that tracks each part of the memory separately.
+The graphs in this server dashboard provide metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab or you can click *Breakdown* to see detailed memory usage metrics.
```

Comment:
This seems partially redundant of the View Server Metrics topic, and in the ORG comments I questioned having the two distinct and in this order.